spec/models/user_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  it 'is invalid without a name' do
+    user = User.new(name: nil)
+    expect(user).to_not be_valid
+  end
+
+  it 'is valid with a non-negative integer posts counter' do
+    user = User.new(name: 'Alice', posts_counter: 0)
+    expect(user).to be_valid
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=equal%20to%20zero.-,Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods%20and%20validations.,-Need%20a%20big), Please add unit tests for the `three_recent_posts` method because you forgot to add them.